ops/scripts/tf_init.sh

```diff
@@ -104,7 +104,7 @@ echo "GITLAB_USERNAME=$GITLAB_USERNAME" >> .init_env_vars
 echo "GITLAB_PRIVATE_TOKEN=$GITLAB_PRIVATE_TOKEN" >> .init_env_vars
 echo "aws_ssh_key=$aws_ssh_key" >> .init_env_vars
 echo "deployment_target=$target_environment" >> .init_env_vars
-echo "backup_file=$backup_file" >> .init_env_vars
+echo "backup_file=../../../../gigadb/app/tools/files-url-updater/sql/$backup_file" >> .init_env_vars
```

Comment:
This change is not required because the user is required to provide the full relative path to the backup file for the `$backup_file` variable when running `tf_init.sh`.